### ISSUE TITLE
Chrony is not found and time was not sync (but I can install manually and fixes it)

### ISSUE THREAD

**Issue by Jacfger** [pop-os/pop#2642]
**Distribution (run `cat /etc/os-release`):**
 ```
NAME="Pop!_OS"
VERSION="22.04 LTS"
ID=pop
ID_LIKE="ubuntu debian"
PRETTY_NAME="Pop!_OS 22.04 LTS"
VERSION_ID="22.04"
HOME_URL="https://pop.system76.com"
SUPPORT_URL="https://support.system76.com"
BUG_REPORT_URL="https://github.com/pop-os/pop/issues"
PRIVACY_POLICY_URL="https://system76.com/privacy"
VERSION_CODENAME=jammy
UBUNTU_CODENAME=jammy
LOGO=distributor-logo-pop-os
```
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**
```
chrony:
  Installed: (none)
  Candidate: 4.2-2ubuntu2
  Version table:
     4.2-2ubuntu2 500
        500 http://apt.pop-os.org/ubuntu jammy/main amd64 Packages
```
(I manually installed it afterwards, but the problem was why was it missing at the first place.)
**Issue/Bug Description:**
tl;dr: the time was not synchronized all in a sudden. And chrony was not installed when I checked it.

I tried to google the issue to solve my PC time not being synchronized. (It's off by few hours after switching from Windows). It hasn't happened in the past and had only occurred until just now. According to other post, Ubuntu uses ```systemd-timesyncd```, but systemctl told me it doesn't exist. Then I heard PopOS uses chrony (https://system76.com/time), but I found that my PopOS doesn't actually had chrony installed. Since I updated my PopOS from 21.04, I wonder if the update didn't include this package (apparently it was only included starting from 22.04 https://system76.com/time). 

**Some update**
I've tried to install it myself and it works. But I don't understand why it stop working at the first place. I'm suspecting maybe one of the updates screws it up? Because
1. My system could sync time before. (so it shouldn't be caused by me upgrading the system from 21.04, or it might idk.)
2. I also remember few days ago my dkpg-configure were all gone suddenly and I had to configure it again for some reason. So maybe some updates screwed it up?

I know I might sound being nitpicking right now but I'm trying to know if the system updates actually messes the system up and had affected other packages or components. 

**Steps to reproduce (if you know):**
No...

**Expected behavior:**
It should be installed without me doing anything. (also the time should sync without me installing anything)

**Other Notes:**

**Comment by jacobgkau** [comment]
`pop-desktop` has a hard dependency on `chrony`: https://github.com/pop-os/desktop/blob/938056f98750487054d86fce6330be344cf0fc95/debian/control 

Can you provide the output of `apt policy pop-desktop chrony` as well as upload a copy of `/var/log/apt/history.log`? (You may need to rename the `.log` file to end with `.txt` for GitHub to allow attaching it.)

**Comment by jacobgkau** [comment]
> I also remember few days ago my dkpg-configure were all gone suddenly and I had to configure it again for some reason. So maybe some updates screwed it up?

I don't understand what `were all gone` means in the context of `dpkg-configure`. I assume you mean you were getting an error message of some kind? What did it say, specifically?

**Comment by Jacfger** [comment]
> `pop-desktop` has a hard dependency on `chrony`: https://github.com/pop-os/desktop/blob/938056f98750487054d86fce6330be344cf0fc95/debian/control
> 
> Can you provide the output of `apt policy pop-desktop chrony` as well as upload a copy of `/var/log/apt/history.log`? (You may need to rename the `.log` file to end with `.txt` for GitHub to allow attaching it.)
chrony:
  Installed: 4.2-2ubuntu2
  Candidate: 4.2-2ubuntu2
  Version table:
 *** 4.2-2ubuntu2 500
        500 http://apt.pop-os.org/ubuntu jammy/main amd64 Packages
        100 /var/lib/dpkg/status
pop-desktop:
  Installed: 1.6.0~1658260090~22.04~e9e6703
  Candidate: 1.6.0~1663192522~22.04~938056f
  Version table:
     1.6.0~1663192522~22.04~938056f 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
 *** 1.6.0~1658260090~22.04~e9e6703 100
        100 /var/lib/dpkg/status
I've installed the chrony myself yesterday. I'm not sure if you can see anything from it.
[history.txt](https://github.com/pop-os/pop/files/9772468/history.txt) Here's the history.log (And also there were multiple backups, if you want it I can attach those as well)

Well, I forgot what I was exactly doing, but probably installing something through apt. 
```libglib2.0-0:i386 depends on zlib1g (>= 1:1.2.2); however: package zlib1g:i386 is not configured yet.``` I saw this error and I fixed it by ```sudo dpkg --configure -a```. Again, I'm not sure if they're related at all in any way.

**Comment by jacobgkau** [comment]
```
pop-desktop:
  Installed: 1.6.0~1658260090~22.04~e9e6703
  Candidate: 1.6.0~1663192522~22.04~938056f
```

The problem is shown here. You have an outdated version of `pop-desktop` installed. Chrony would have been installed automatically when you upgraded, because `938056f` is the version that pulls in Chrony as a dependency-- that version was released on September 15th, nearly a month ago, at the same time as the systemd update that removed systemd-timesyncd: https://github.com/pop-os/repo-release/pull/173 

When is the last time you installed updates on your system? Please provide the output of `sudo apt update` and `sudo apt full-upgrade` to show why `pop-desktop` is not being upgraded. If the output indicates packages are being held back, then please also provide the output of `sudo apt install pop-desktop=1.6.0~1663192522~22.04~938056f`.

**Comment by Jacfger** [comment]
```
chrony:
  Installed: 4.2-2ubuntu2
  Candidate: 4.2-2ubuntu2
  Version table:
 *** 4.2-2ubuntu2 500
        500 http://apt.pop-os.org/ubuntu jammy/main amd64 Packages
        100 /var/lib/dpkg/status
pop-desktop:
  Installed: 1.6.0~1663192522~22.04~938056f
  Candidate: 1.6.0~1663192522~22.04~938056f
  Version table:
 *** 1.6.0~1663192522~22.04~938056f 1001
       1001 http://apt.pop-os.org/release jammy/main amd64 Packages
        100 /var/lib/dpkg/status
```

```apt full-upgrade``` sucessfully upgraded PopOS for me though, what does that mean? (The log was quite length and I don't see anything interesting, so I posted ```apt policy chrony pop-desktop``` instead). So somehow an upgrade remove systemd-timesyncd but it doesn't install chrony for me? Would that be the problem of me installing updates through pop-shop (since it allows me to upgrade individual packages easily)?
